src/wp-includes/functions.php

```diff
@@ -8582,3 +8582,93 @@ function is_php_version_compatible( $required ) {
 function wp_fuzzy_number_match( $expected, $actual, $precision = 1 ) {
 	return abs( (float) $expected - (float) $actual ) <= $precision;
 }
+
+/**
+ * Get supported data types for meta and option values.
+ *
+ * This is a subset of the PHP data types that can be used for
+ * meta and option values. See the documentation for PHP's 'settype()'.
+ *
+ * @since 6.3.0
+ *
+ * @return array List of supported value types.
+ */
+function wp_get_db_value_types() {
```

Comment:
Is it worth having a separate function for this array, rather than just defining it in `wp_settype_to_value_from_db()`? What are the other use cases you have in mind?